core/deployment/src/main/java/io/quarkus/deployment/builditem/CapabilityBuildItem.java

```diff
@@ -24,6 +26,12 @@
  */
 public final class CapabilityBuildItem extends MultiBuildItem {
 
+    private static volatile Logger log;
+
+    private static Logger getLog() {
+        return log == null ? log = Logger.getLogger(CapabilityBuildItem.class) : log;
```

Comment:
Why do you do that? We are at deployment time, it's not that big of deal to have a static logger, or is it?

Reply:
At this point it's used only when the `CapabilityBuildItem(String)` is called, which is practically never. Otherwise, you are right.